### PR TITLE
Fix #95

### LINKED ISSUE #95: Makefile is not working correctly
When i run `make rpm` i get following output:

>The archive is in yum-3.4.3.tar.gz
cp: cannot create regular file 'build/SOURCES/': Not a directory
Makefile:137: recipe for target 'srpm' failed
make: *** [srpm] Error 1

### PR DESCRIPTION
**Pull request by dnagl**
This commit should fix issue #95

### CODE REVIEW

**Comment by dmnks** [comment]
Hi David,
Are you sure this was the culprit? Doesn't `/path/{A,B,C}` expand to `/path/A`, `/path/B` and `/path/C` anyway?

**Comment by dnagl** [comment]
I know, I tried the command manually on the commandline and there this worked fine.
I tried this on several machines with different OS and different environments. But I was not able to build this. The command always creaed this folder `build/SOURCES,SRPMS,RPMS` and so the following steps which are trying to work in the generated directories do not work.

**Comment by dmnks** [comment]
OK, it seems brace expansions are [not POSIX-compliant](https://unix.stackexchange.com/a/92829) (although `bash` and most other shells do support them).

I'm fine with the patch then. Before I merge it, though, please also do the same correction under the `clean:` target where we do `rm -f $(BUILDDIR)/{SOURCE,SRPMS,RPMS}/*`.

**Comment by dnagl** [comment]
@dmnks I added the changes in the `clean:` target.

File: Makefile [hunk 1]
```diff
@@ -19,7 +19,7 @@ all: subdirs
 
 clean:
 	rm -f *.pyc *.pyo *~ *.bak
-	rm -f $(BUILDDIR)/{SOURCES,SRPMS,RPMS}/*
+	rm -f $(BUILDDIR)/SOURCES/* $(BUILDDIR)/SRPMS/* $(BUILDDIR)/RPMS}/*
```
Comment (dmnks) [inline Makefile]: There's a typo: the brace at the end should also be removed.

**Review by dmnks — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (dnagl) [reply Makefile]: Thanks for the hint. The typo is now fixed.

**Comment by dmnks** [comment]
Merged, thanks!

**Pull request merged**